prometheus-exporters/vrops-exporter/alerts/datastore.alerts

```diff
@@ -51,12 +51,12 @@ groups:
       tier: vmware
       service: storage
       context: "vrops-exporter"
-      meta: "Datastore {{ $labels.datastore }} utilization > 80% ({{ $labels.datacenter }}, {{ $labels.vccluster }}, {{ $labels.hostsystem }})"
+      meta: "Datastore {{ $labels.datastore }} utilization > 80% ({{ $labels.vcenter }}, {{ $labels.vccluster }}, {{ $labels.hostsystem }})"
```

Comment:
Please either go with a `.` after the general statement or without it. Some examples do have a `.` at the end, some don't.